aerogear/aerogear-unifiedpush-server
# Gcm topics

This is a bit of a big one so it needs lots of eyes and lots of commentary.  I hope.

 # How to test :
* Build the project. 
* Build the https://github.com/aerogear/aerogear-android-push/pull/50 for AGDROID
  *
* Configure a push variant etc etc in UPS
* Configure a sample application (I used the hello world)
* In the sample application add a category "topic" (or anything you want) to the Push Registrar config
* In UPS sent a message to the device. In the UPS log you will see Sending payload for [1] devices to GCM
* Now send a message but set the category to "topic" (or whatever you set in step *)
* In UPS you should see Sending payload for [1] topics to GCM
* You should also see both messages in the app.
